doc/source/example_notebooks/IS2_data_read-in.ipynb

```diff
@@ -63,9 +63,8 @@
    "metadata": {},
```

Comment:
<p>Suggestion:</p><p>glob will not by default search all of the subdirectories for matching filepaths, but it has the ability to do so. If you would like to search recursively, you can achieve this by:</p><p> </p><ol><li>passing the <code>recursive=True</code> argument into <code>glob_kwargs</code> (shown below)</li><li>use <code>/**/</code> in the filepath to match any level of nested folders (not shown below)</li><li>using glob directly to create a list of filepaths (shown below)</li></ol><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/icesat2py/icepyx/pull/444/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>